--- fortunes-pl-2.0_2002.09.12/komputery~	Mon Mar 31 22:54:56 2003
+++ fortunes-pl-2.0_2002.09.12/komputery	Mon Mar 31 22:54:56 2003
@@ -61,7 +61,7 @@
 			--- Richard Letts w Monastery
 %
    From: root@emily.tampa.advantis.com (John R. Campbell)
-   Jako SysAdmin MOGÊ czytaæ twoje maile, ale NIE nudze siê
+   Jako SysAdmin MOGÊ czytaæ twoje maile, ale NIE nudzê siê
    a¿ tak.
 %
    From: ftit@engin.umich.edu (Sergej Roytman)
@@ -105,13 +105,13 @@
 			    <ISBN:0-201-89685-0>]
 %
    From: Tom Christiansen <tchrist@mox.perl.com>
-   "Nale¿y siê zapisaæ na listê dyskusyjna po to, by nie otrzymywaæ
+   "Nale¿y siê zapisaæ na listê dyskusyjn± po to, by nie otrzymywaæ
    ca³ego tego ¶miecia, które widzi siê na grupach dyskusyjnych".
 			--- Dennis Ritchie
 %
    From: Matthias Neeracher <neeri@iis.ee.ethz.ch>
    "U¿ywanie szyfrowania w Internecie przypomina wynajêcie samochodu
-   opancerzonego, aby przewie¼æ informacje o karcie kredytowej od
+   opancerzonego, aby przewie¼æ informacjê o karcie kredytowej od
    kogo¶, kto ¿yje w kartonie, do kogo¶, kto mieszka na ³awce w parku."
 			--- Gene Spafford
 %
@@ -121,7 +121,7 @@
    From: Jan Dockx <Jan.Dockx@cs.kuleuven.ac.be>
    "Istnieje anegdota o pewnym matematyku, który mia³ powiedzieæ, ¿e
    1000 szympansów siedz±cych przed 1000 maszyn do pisania
-   po 1000 lat napisa³oby wszystkie dzie³a Shakespeara.
+   po 1000 lat napisa³oby wszystkie dzie³a Shakespeare'a.
    Ale wystarczy spojrzeæ na Internet, by zobaczyæ, ¿e tak nie jest"
    
 			--- Kurt Vonnegut o Internecie w "Vonnegut's
@@ -409,7 +409,7 @@
 
    From: bobb@vice.ICO.TEK.COM (Bob Beauchaine)
    C: jêzyk, który ³±czy moc asemblera z elastyczno¶ci± asemblera.
-   Prawdziwi przyjaciele nie pozwalaj± przyjacio³om u¿ywac Unixa.
+   Prawdziwi przyjaciele nie pozwalaj± przyjacio³om u¿ywaæ Unixa.
 %
    From: nevries@accucx.cc.ruu.nl (Nico E. de Vries)
    Niestety, aktualna generacja programów mailowych nie posiada
@@ -434,8 +434,8 @@
    Nawet Unix mo¿e nie byæ a¿ tak z³y dla Prawdziwych Programistów,
    jak by³ kiedy¶. Ostatnia wersja Unixa ma potencja³ systemu 
    operacyjnego godnego ka¿dego Prawdziwego Programisty -- dwa ro¿ne
-   i subtelnie niekompatybilne intrefajsy u¿ytkownika, tajemniczy
-   i skomplikowany driver terminala, pamieæ wirtualna. Je¶li zignoruje
+   i subtelnie niekompatybilne interfejsy u¿ytkownika, tajemniczy
+   i skomplikowany driver terminala, pamiêæ wirtualn±. Je¶li zignoruje
    siê fakt, ¿e jest strukturalne, nawet C mo¿e byæ zaakceptowane
    przez Prawdziwego Programistê: w koñcu nie ma tam sprawdzania
    typów, nazwy zmiennych maj± siedem (osiem? dziewiêæ?) znaków,
